runtime/port/win32/j9sock.c

```diff
@@ -158,7 +158,7 @@ findError (int32_t errorCode)
 {
 	switch (errorCode)
 	{
-		case WSANOTINITIALISED:
+		case WSANOTINITIALIZED:
```

Comment:
`WSANOTINITIALISED ` is actually a Windows socket error code.

https://msdn.microsoft.com/en-us/library/windows/desktop/ms740668(v=vs.85).aspx

Reply:
Fixed in 82cbf35. Launched a new build now. Thanks for catching this!